src/pages-and-resources/progress/Settings.jsx

```diff
@@ -26,17 +26,23 @@ function ProgressSettings({ intl, onClose }) {
       onSettingsSave={handleSettingsSave}
     >
       {
-        ({ handleChange, handleBlur, values }) => (
-          <FormSwitchGroup
-            id="enable-progress-graph"
-            name="enableProgressGraph"
-            label={intl.formatMessage(messages.enableGraphLabel)}
-            helpText={intl.formatMessage(messages.enableGraphHelp)}
-            onChange={handleChange}
-            onBlur={handleBlur}
-            checked={values.enableProgressGraph}
-          />
-        )
+        ({ handleChange, handleBlur, values }) => {
+          let formSwitch = null;
+          if (getConfig().ENABLE_PROGRESS_GRAPH_SETTINGS === 'true') {
```

Comment:
just wanted to confirm if that will contain string value or bool?

Reply:
Yes, env values are always string .